tests/move_architecture/test_architecture.py

```diff
@@ -86,10 +49,10 @@ def test_backend_configuration_adonis(adonis_architecture):
     assert backend.target.physical_qubits == [0, 1, 2, 3, 4]
     assert set(backend.target.operation_names) == {'r', 'id', 'cz', 'measure'}
     assert [f'{o.name}:{o.num_qubits}' for o in backend.target.operations] == [
```

Comment:
Is the ordering here deterministic? Why did the ordering change?

Reply:
It is deterministic, it comes from the order in which instructions are added to `target` in IQMBackendBase. The changes related to that were taken from Ville's https://github.com/iqm-finland/qiskit-on-iqm/pull/123.